src/xml/report.html.xsl

```diff
@@ -1,62 +1,122 @@
 <?xml version="1.0" encoding="ISO-8859-1"?>
 
-<xsl:stylesheet 
-  xmlns:xsl="http://www.w3.org/1999/XSL/Transform" 

```

Comment:
In general white space changes and formatting changes should be separate from code/content changes.

https://github.com/zaproxy/zaproxy/wiki/DevGuidelines#formatting